modAionImpl/src/org/aion/zero/impl/sync/msg/ResTxReceipts.java

```diff
@@ -0,0 +1,75 @@
+package org.aion.zero.impl.sync.msg;
+
+import com.google.common.base.Preconditions;
+import org.aion.p2p.Ctrl;
+import org.aion.p2p.Msg;
+import org.aion.p2p.Ver;
+import org.aion.rlp.RLP;
+import org.aion.rlp.RLPElement;
+import org.aion.rlp.RLPList;
+import org.aion.zero.impl.sync.Act;
+import org.aion.zero.impl.types.AionTxInfo;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+/** Response for transaction receipts request */
+public class ResTxReceipts extends Msg {
```

Comment:
please rename this to `ResponseReceipts`; same for the handler and tests classes